WifiART/src/userinterface/LciView.java

```diff
@@ -3,14 +3,7 @@
 import structs.AltitudeType;
 import structs.MapDatum;
 
-import javax.swing.BorderFactory;
-import javax.swing.Box;
-import javax.swing.BoxLayout;
-import javax.swing.JCheckBox;
-import javax.swing.JComboBox;
-import javax.swing.JLabel;
-import javax.swing.JPanel;
-import javax.swing.JTextField;
+import javax.swing.*;
```

Comment:
Generally using a * is considered bad practice as you pull in everything. By using an itemized list, you are ensuring you only pull in what you need.

Reply:
Thanks, I typically get rid of the star if I see it but sometimes the IDE automatically adds it when I add new imports.